.github/workflows/publish_npm.yml

```diff
@@ -35,6 +45,26 @@ jobs:
           registry: https://registry.npmjs.org/
           tag: ${{ steps.nbgv.outputs.PrereleaseVersion == '' && 'latest' || 'preview' }}  # Assign a 'preview' tag to versions end with '-preview'. Otherwise, assign a 'latest' tag to the latest release.
           access: public
+      - name: Publish 'dbos-cloud' release to npm
```

Comment:
I think we can create a similar "publish test package to npm" for dbos-cloud and dbos-openapi. This way, we will first test if the publish command works before merging.

Reply:
Yes, as long as any cut-n-paste errors are the same in both, you'll know something.